doc/cookbook/source/examples/classifier/mkl_binary_class.rst

```diff
@@ -0,0 +1,59 @@
+==================================================
+Multiple Kernel Learning for Binary Classification
+==================================================
+
+Multiple kernel learning (MKL) is about using a combined kernel i.e. a kernel consisting of a linear combination of arbitrary kernels over different domains.
+
+.. math::
+
+    f({\bf x})=\text{sign} \left(\sum_{i=0}^{N-1} \alpha_i k({\bf x}, {\bf x_i})+b\right)
+
+where :math:`{\bf x_i},{i = 1,...,N}` are labeled training examples where label is :math:`y_i \in {±1}`, :math:`\bf x` is the new data point, :math:`k` is a kernel function and :math:`b` is bias.
+
+One could make a combination of kernels like:
+
+.. math::
+
+    {\bf k}(x_i,x_j)=\sum_{k=0}^{K} \beta_k {\bf k_k}(x_i, x_j)
+
+where :math:`\beta_k > 0` and   :math:`\sum_{k=0}^{K} \beta_k = 1`
+
+Optimal :math:`\alpha_i`, :math:`\beta` and :math:`b` are determined via training.
+
+See :cite:`sonnenburg2006large` for more details.
+
+-------
+Example
+-------
+
+Imagine we have files with training and test data. We create CDenseFeatures (here 64 bit floats aka RealFeatures) and :sgclass:`CBinaryLabels` as
+
+.. sgexample:: mkl_binary_class.sg:create_features
+
+Then we create an instance of :sgclass:`CPolyKernel`, initialize it with training data and extract a kernel matrix.
+
+.. sgexample:: mkl_binary_class.sg:create_kernel
+
+We create an instance of :sgclass:`CCombinedKernel` by using :sgclass:`CCustomKernel` and :sgclass:`CPolyKernel`. In order to initialize :sgclass:`CCombineKernel` with features, we also need to create an object of :sgclass:`CCombinedFeatures`.
+
+.. sgexample:: mkl_binary_class.sg:create_combined_train
+
+We create an object of :sgclass:`CMKLClassification`, provide the combined kernel and labels before training it.
+
+.. sgexample:: mkl_binary_class.sg:train_mkl
+
+As shown above for training data, we similarly make a :sgclass:`CCombinedKernel` for testing data.
```

Comment:
@karlnapf or i can also remove `CustomKernel` and only keep `PolyKernel` and `GaussianKernel`. This way I can just reuse the old `CombinedKernel` without any problems.